packages/terra-base/src/Base.jsx

```diff
@@ -68,19 +68,16 @@ class Base extends React.Component {
       ...customProps
     } = this.props;
 
-    const childComponent = (
-      <div {...customProps}>
-        {children}
-      </div>
-    );
-
     const messages = Object.assign({}, this.state.messages, customMessages);
 
-    if (locale === undefined) return childComponent;
+    if (locale === undefined) {
+      return (<div {...customProps}>{children}</div>);
+    }
+
     if (!this.state.areTranslationsLoaded) return <div>{this.props.translationsLoadingPlaceholder}</div>;
     return (
-      <I18nProvider locale={this.state.locale} messages={messages}>
-        {childComponent}
+      <I18nProvider locale={this.state.locale} messages={messages} {...customProps}>
```

Comment:
Move this customProps to the start as well.

Reply:
Updated https://github.com/cerner/terra-core/commit/09df497d2f4ae7b6867cfccb9ce98e6d7f1bef7c